docs/developers-clear-containers-install.md

```diff
@@ -57,8 +58,10 @@ Both projects ship ``cc-proxy`` and they are not compatible with each other.
    ```bash
    $ go get -d github.com/clearcontainers/runtime
    $ go get -d github.com/clearcontainers/proxy
+   $ go get -d github.com/clearcontainers/agent
```

Comment:
nitpick - you have Agent after Shim in the previous list - the opposite way around to here.

Reply:
Fixed.